From 3ed368f5a084591ce93df04af53cec3912c313ee Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.liu@freescale.com>
Date: Tue, 8 Jan 2013 16:26:07 +0800
Subject: [PATCH 381/435] ENGR00239215 VPU: Fix a potential build break

This patch fix a potential build break in Android:
==================================================
In file included from external/linux-test/test/
mxc_vpu_test/main.c:21:0:
In function 'fgets',
inlined from 'parse_config_file' at
external/linux-test/test/mxc_vpu_test/main.c:223:8:
bionic/libc/include/stdio.h:554:30: error:
call to '__fgets_too_big_error' declared with
attribute error: fgets called with size bigger than buffer
make: *** [out/target/product/sabresd_6dq/obj/EXECUTABLES/
mxc-vpu-test_intermediates/main.o] Error 1
make: Leaving directory `/home/b17645/repos/AndroidJB'
==================================================

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 test/mxc_vpu_test/main.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/mxc_vpu_test/main.c b/test/mxc_vpu_test/main.c
index 8366828..2e57889 100644
--- a/test/mxc_vpu_test/main.c
+++ b/test/mxc_vpu_test/main.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2012 Freescale Semiconductor, Inc.
+ * Copyright 2004-2013 Freescale Semiconductor, Inc.
  *
  * Copyright (c) 2006, Chips & Media.  All rights reserved.
  */
@@ -213,7 +213,7 @@ int
 parse_config_file(char *file_name)
 {
 	FILE *fp;
-	char line[128];
+	char line[MAX_PATH];
 	char *ptr;
 	int end;
 
-- 
1.8.0

